src/bielewicz-jacek/Fsd.Bielewicz.Jacek.Cs(1projekt)/Fsd.Bielewicz.Jacek.Cs1/Tools/Counting.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Bielewicz.Jacek.Cs1.Tools
+{
+    public class Counting : View
+    {
+        public void Count(string[] args)
+        {
+            int countUpper = 0, countLower = 0, countDigit = 0, j, numberOfIndexAtEachIteration = 0, argumentsLength = args.Length;
+            string sentence;
+            
+            void CountDigit()
```

Comment:
funkcja w funkcji raczej jest niezalecana - jest ona możliwa dopiero od C#7. Możesz to zrobić inaczej